### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by ShaheerCH**
Add linters

### CODE REVIEW

**Review by ShaheerCH — COMMENTED** [review]
Review made the details are below

File: index.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+    <link rel="stylesheet" href="css/style.css" />
   </head>
   <main id="main">
-  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
     <h1 id="title">Dr. Norman Borlaug</h1>
     <p>The man behind green revolution</p>
     <figure id="img-div">
```
Comment (ShaheerCH) [inline index.html:16]: It suggest that you add sementic header tags. It would make the code easier to follow

File: html-css/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# HTML & CSS3 Course
```
Comment (ShaheerCH) [inline html-css/README.md:1]: Great Readme file. It would really be helpful for reviewer if the readme was about the website.

File: index.html [hunk 3]
```diff
@@ -21,31 +24,106 @@ <h1 id="title">Dr. Norman Borlaug</h1>
         how to increase wheat yields.
       </figcaption>
     </figure>
-    <section id="tribute-info">
```
Comment (ShaheerCH) [inline index.html:24]: Kindly Add all the visible items in body tags

File: css/style.css [hunk 4]
```diff
@@ -1,24 +1,34 @@
...
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+.container-fluid {
+  padding: 10px;
+}
+
+body {
```
Comment (ShaheerCH) [inline css/style.css:16]: this styling has no corresponding body tag. I suggest you add body tags in html for this styling to appear

**Review by ShaheerCH — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ShaheerCH👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [x] Please make sure the README file looks more professional. Feel free to use this [template ](https://github.com/microverseinc/readme-template)but customize it for your project.
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by aimemalaika — COMMENTED** [review]
Hi @ShaheerCH , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**